server/schema/questions/createQuestions.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  required: ['type', 'name', 'problem_statement'],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['mcq', 'subjective'],
+    },
+    name: {
+      type: 'string',
+    },
+    problem_statement: {
+      type: 'string',
+    },
+    input_format: {
+      type: 'string',
+    },
+    output_format: {
+      type: 'string',
+    },
+    constraints: {
+      type: 'string',
+    },
+    options: {
+      type: 'string',
+      pattern: '^\\w+(?:(?:,\\w+)+|(?:\\s\\w+)+)$',
+    },
+    correct: {
+      type: 'number',
+    },
+    difficulty: {
+      type: 'string',
+      enum: ['easy', 'medium', 'hard'],
```

Comment:
easy will be assigned 0....but we want 0 for `not_specified`

Reply:
Let me make it optional, and change that in sql as well so that means if its NULL then that means not specified

Reply:
i think it will be bothersome changing sql

Reply:
Yeah, changing in SQL to allow NULL will do. No need to change in Schema, then